css-two-column-layout/styles.css

```diff
@@ -0,0 +1,66 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  font-family: 'Karla', sans-serif;
+  max-width: 1100px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.column-half {
+  flex-basis: 50%;
+  display: flex;
```

Comment:
You shouldn't make your `column` elements into flex elements like this. This would be a case where you should add a `row` element as a child of this column, then make the content a child of that row.